examples/drupal_loadtest.rs

```diff
@@ -72,13 +72,13 @@ fn main() {
 }
 
 /// View the front page.
-fn drupal_loadtest_front_page(client: &mut GooseClient) {
-    let response = client.get("/");
+async fn drupal_loadtest_front_page<'r>(client: &'r mut GooseClient) {
```

Comment:
It turns out the lifetime is not needed when using the macro to call this:

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=804f51aa07a240299de50b5211506a5c

So this can be simplified back to:

```
async fn drupal_loadtest_front_page(client: &mut GooseClient) {
```

Reply:
Oh right. Sneaky! This is because async fns can do lifetime elision for you.

I guess that settles it in favour of a `task!()`-style macro then. I'm not in the mood for writing lifetime elision in an attribute macro.

Reply:
This is really exciting guys. I love it when a plan comes together :D.

Reply:
Agreed!! Thanks for all the time and effort!